hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -393,7 +405,7 @@ def get_dashboard_system_status() -> DashboardSystemStatus:
     )
 
 
-def get_hash_count() -> t.Dict[str, int]:
+def get_signal_hash_count() -> t.Dict[str, int]:
```

Comment:
blocking: Seems like we call this function a bunch and its quite expensive. Can we memoize it with `@cache`?

Reply:
agree but punting for now @Dcallies thoughts on @lru_cache in another PR maybe?

Reply:
I'm good with punting to a future PR